packages/filesystem/src/browser/style/file-dialog.css

```diff
@@ -87,17 +102,56 @@
     left: 41px;
 }
 
+.dialogContent .theia-NavigationUp {
+    left: 61px;
```

Comment:
There are a lot of very specific pixel values here. Do you see any way to get to a more ergonomic solution (grid, flex) to the placement of these elements?

Reply:
I had thought about this too, but in the spirit of how the file dialog currently exists and is styled I decided to retain the absolute positioning as to not introduce too many new changes to the CSS.